dora/underfs/hdfs/pom.xml

```diff
@@ -135,6 +129,17 @@
   </build>
 
   <profiles>
+    <profile>
```

Comment:
i had to sort this after we require poms to be sorted. No idea why there wasn't this problem before I touched it...